backend/app/configs/data_saver/jsonl_saver.yaml

```
@@ -4,4 +4,5 @@ defaults:
 
 name: jsonl_saver
 source: $kafka
-jsonl_file_path: ${oc.env:ROOT_PATH}/app/data_layer/database/db/jsonl_saver.jsonl
\ No newline at end of file
+jsonl_file_path: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)